README.md

```diff
@@ -88,6 +88,21 @@ Under the hood, Rider launches the Core Tools with the `--dotnet-isolated-debug`
 
 You can place a breakpoint in any function, and inspect your code as it is running. Note that [debugging startup code may timeout (#434)](https://github.com/Azure/azure-functions-dotnet-worker/issues/434).
 
+## Running E2E Tests
+
+### Requirements
+
+- [Powershell 7](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.2)
+- [CosmosDb Emulator](https://docs.microsoft.com/en-us/azure/cosmos-db/local-emulator?tabs=ssl-netstd21)
+- [Azure Storage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator)
```

Comment:
Is this a valid requirement? The setup should install Azurite, correct?

Reply:
I think it's always worth noting all the "things" that will be used / are needed for the E2E tests to run. You can add a note that the script will download this for you 🤷‍♀️

Reply:
My concern is that listing those as requirements at the beginning, with no additional information, will likely push people following this to go install those components before proceeding, even though this is handled by the script. It's fine to mention what the script will do, but I wouldn't position this as a requirement/pre-requisite to move forward with the process, which is what this implies.